examples/plotting/file/sparkline.py

```diff
@@ -0,0 +1,73 @@
+import pandas as pd
+from bokeh.charts import TimeSeries
+from bokeh.models import (
+    Legend,
+    Circle,
+    HoverTool,
+    ColumnDataSource,
+    GlyphRenderer
+)
+from bokeh.palettes import brewer
+
+TOOLS = "hover"
+palette = brewer["Blues"][3]
+
+apple = pd.read_csv(
+    "http://ichart.yahoo.com/table.csv?s=AAPL&a=0&b=1&c=2000&d=0&e=1&f=2010",
+    parse_dates=['Date']
+)
+
+# Basic sparkline
+
+sparkline = TimeSeries(
+    apple[['Adj Close', 'Date']], index='Date',
+    title="", tools=TOOLS, height=100, palette=palette,
+    ylabel='', xlabel='', xgrid=None, ygrid=None,
+    filename="sparkline.html"
+)
+sparkline.add_layout(
+    Legend(
+        legends=[('Apple', None), ],
```

Comment:
Also, bokehjs shouldn't fail this miserable way. We should either not allow `None` in Python API in this case (i.e. make glyph's instance required) or deal with `null` in bokehjs.